service-runner/src/main/resources/common/docker-compose.yaml

```diff
@@ -36,6 +36,16 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
+  custody-server:
+    image: stellar/anchor-platform:edge
+    build:
+      context: ../../../../../integration-tests/src/test
```

Comment:
Is this avoidable?

Reply:
Fixed in https://github.com/stellar/java-stellar-anchor-sdk/pull/1082